sechub-scan/src/test/java/com/mercedesbenz/sechub/domain/scan/WebConfigBuilderStrategyTest.java

```diff
@@ -195,245 +191,10 @@ public void form_script_login_data_transfered() {
         assertEquals(null, action.getValue());
     }
 
-    @Test
-    public void excludes_no_slash_infront_single() {
-        /* prepare */
-        List<String> excludes = new LinkedList<>();
-        excludes.add("contact.html");
-
-        String json = createExcludesJson(excludes);
```

Comment:
It is good that the validation is removed from the builder strategy. 
But the tests here did something which is not inside the validator test:
They use SECHUB_CONFIG.fromJson(json) -> the excludes/includes were tested with a json text.

`SecHubConfigurationModelTest` does do this also, but not with the new asterisks.
Please do me a favor and add one example with asterisk to `SecHubConfigurationModelTest`, so it is clear that the model can be created when this is inside (should always work, but it is better to have it inside the tests as well)